malio/src/main/java/com/github/nalukit/malio/shared/annotation/field/ValidateItem.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2023 Frank Hossfeld, Philipp Kohl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.nalukit.malio.shared.annotation.field;
+
+import java.lang.annotation.Annotation;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>The annotation tells Malio to validate an item
+ * of a Collection or Array</p>
+ *
+ * <p>This annotation can only be used on fields of type <b>Array, Collection</b>.</p>
+ *
+ * @author Frank Hossfeld, Philipp Kohl
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface ValidateItem {
```

Comment:
Does this belong in this PR?

Reply:
No ... and will be removed with the next PR. In this PR it's only dead code.